crypto/x509/x509_lu.c

```diff
@@ -310,6 +310,19 @@ X509_OBJECT *X509_STORE_CTX_get_obj_by_subject(X509_STORE_CTX *ctx,
     return ret;
 }
 
+
+static int x509_object_up_ref_count(X509_OBJECT *a)
```

Comment:
I'd rename it to `x509_object_up_ref()`. Now that it's no longer public there's no need for it to be different from all the others.

Reply:
doh!  yup.

Reply:
Hrm. Pondering a bit more, maybe it's wrong. X509_OBJECT does not have a refcount itself, so we don't really bump its refcount. That's probably the reason for the strangely inconsistent name.

Reply:
Not gonna guess on the rationale for the name :). How about just `up_ref_count` ?

Reply:
Do I detect sarcasm? :). Sure, I'll revert the name change.

Reply:
Do I detect sarcasm? :) okay not touching.

Reply:
The boundaries between sarcasm, cynicism, irony and exasperation are quite fluid, especially when I deal with OpenSSL legacy. A little bit of all of them, I suppose.

To the PR at hand: I don't mind calling it whatever you like, but my opinion is that keeping it at `x509_object_up_ref_count()` rather than the up_ref you have now is best. Just choose what you like and I'll approve later today. Nothing cared so far, but there's a lot of small perl modules still queued up.